packages/models/src/authorization/protobufConverterFromV2.ts

```diff
@@ -17,6 +17,7 @@ const fromKeyUseV2 = (input: KeyUseV2): KeyUse => {
 
 export const fromKeyV2 = (input: KeyV2): Key => ({
   ...input,
+  userId: input.userId ? unsafeBrandId<UserId>(input.userId) : undefined,
```

Comment:
in V2 `userId` is mandatory
```suggestion
  userId: input.userId,
```